browser/ui/brave_actions/brave_action_icon_with_badge_image_source.cc

```diff
@@ -8,7 +8,7 @@
 #include <algorithm>
 
 #include "base/strings/utf_string_conversions.h"
-#include "brave/browser/ui/brave_actions/constants.h"
+#include "brave/common/extensions/constants.h"
```

Comment:
do we need a dep added for any of these?

Reply:
For `brave/common/extensions/constants.h`, there is only the `.h` that contains 2 const values, so there is nothing else to depend on.
`//brave/browser/extensions` and `//brave/browser/ui` already have a dep on `//extensions/browser`